server.py

```diff
@@ -1025,7 +1137,7 @@ def commit_to_branch(repo, branch, code, rep_dir, filename, commit_msg, file_sha
     if file_sha:
         data["sha"] = file_sha
 
-    response = github.put(f"repos/{repo}/contents/{rep_dir}/{filename}", data=data)
+    response = github_call("PUT", f"repos/{repo}/contents/{rep_dir}/{filename}", data=data)
```

Comment:
I got an error, maybe from here: `github_call() got an unexpected keyword argument 'data'`

Reply:
Yes it should be `params`. Thanks for spotting that. It's wrong in more than one place, actually. As I mentioned in the PR description I wasn't able to back-test all of the old functionality. Sorry about that. I'll push a fix in a minute.

Reply:
Try it again now.